scripts/docker/README.md

```diff
@@ -45,6 +46,41 @@ export STAGING_DIR=~/tmp/tmp_build_docker_image/
 sudo bash build_docker.sh -e ${GITHUB_TAG} -p -d ${STAGING_DIR}
 ```
 
+#### Create GATK4 docker image and upload into GCR
+
+This will attempt to build and automatically upload the docker image with a tag following this format: `us.gcr.io/broad-dsde-methods/broad-gatk-snapshots/gatk-remote-builds:<USERNAME>-<GITHUB_TAG>-<GIT_HASH>`
```

Comment:
Make it clear that the build happens in the cloud, and this is a good option for people having trouble building the image locally.